client/a8c-for-agencies/sections/woopayments/primary/woopayments-dashboard/index.tsx

```diff
@@ -161,21 +145,16 @@ const WooPaymentsDashboard = () => {
 				value={ {
 					woopaymentsData,
 					isLoadingWooPaymentsData,
-					sitesWithPluginsStates: sitesWithPluginsStatesSorted,
+					sitesWithPluginsStates: sortedSitesWithWooPayments,
 				} }
 			>
 				<LayoutTop isFullWidth={ isFullWidth }>
-					{ !! sitesWithPluginsStates.length && <MissingPaymentSettingsNotice /> }
+					{ !! allSitesWithWooPayments.length && <MissingPaymentSettingsNotice /> }
 					<LayoutHeader>
 						<Title>{ title }</Title>
 						<Actions>
 							<MobileSidebarNavigation />
 							<div className="woopayments-dashboard__actions">
-								{ isInProgress && (
-									<div className="woopayments-dashboard__spinner">
-										<Spinner /> { translate( 'Loading and refreshing data' ) }
-									</div>
-								) }
```

Comment:
@tiagonoronha I have removed the code that we had to show a loading indicator, as we no longer have async fetching here.